src/amber/cli/templates/app.cr

```diff
@@ -24,5 +32,22 @@ module Amber::CLI
     private def generate_database_name_base
       @name.gsub('-', '_')
     end
+
+    def fetch_author
+      return "[your-name-here]" unless system(WHICH_GIT_COMMAND)
+      `git config --get user.name`.strip
+    end
+
+    def fetch_email
+      return "[your-email-here]" unless system(WHICH_GIT_COMMAND)
+      `git config --get user.email`.strip
```

Comment:
User may not also have `user.email` and `user.name` configured, so `fetch_email` and `fetch_author` will return an empty string (instead of `"[your-email-here]"`).

Reply:
Well, we already used this without major problems for all crystal projects until now, check: https://github.com/crystal-lang/crystal/blob/master/src/compiler/crystal/tools/init.cr

Reply:
I didn't say it has major problems, it just doesn't work in all cases proreply and easy to fix.

Reply:
@veelenga Done! Thank you for your suggestion! 👍